fe/fe-core/src/main/java/com/starrocks/sql/analyzer/Analyzer.java

```diff
@@ -97,6 +98,12 @@ public Void visitShowStatement(ShowStmt statement, ConnectContext session) {
             return null;
         }
 
+        @Override
```

Comment:
It is a good design to unify Drop related statement into one DropAnalyzer. But we don't need a separate visitDropStatement, it is enough to handle it in visitDropTableStmt